wpilibcExamples/src/main/cpp/examples/RapidReactCommandBot/cpp/RapidReactCommandBot.cpp

```diff
@@ -12,16 +12,13 @@
 
 void RapidReactCommandBot::ConfigureBindings() {
   // Automatically run the storage motor whenever the ball storage is not full,
-  // and turn it off whenever it fills.
-  frc2::Trigger([this] {
-    return m_storage.IsFull();
-  }).WhileFalse(m_storage.RunCommand());
+  // and turn it off whenever it fills. Uses subsystem-hosted trigger to
+  // improve readability and make inter-subsystem communication easier.
+  m_storage.HasCargo.WhileFalse(m_storage.RunCommand());
```

Comment:
Any reason not to put this in the subsystem itself?

Reply:
I think it should still be recommended for the actual bindings to live in RobotContainer for organizational reasons (why dont we keep all constants in their respective subsystem classes?) and to again enable the easier interop.